runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/FakeWindmillServer.java

```diff
@@ -280,7 +329,20 @@ public Windmill.GlobalData requestGlobalData(Windmill.GlobalDataRequest request)
       }
 
       @Override
-      public void refreshActiveWork(Map<String, List<KeyedGetDataRequest>> active) {}
+      public void refreshActiveWork(Map<String, List<KeyedGetDataRequest>> active) {
+        Windmill.GetDataRequest.Builder builder = Windmill.GetDataRequest.newBuilder();
+        for (Map.Entry<String, List<KeyedGetDataRequest>> entry : active.entrySet()) {
+          builder.addRequests(
+              ComputationGetDataRequest.newBuilder()
+                  .setComputationId(entry.getKey())
+                  .addAllRequests(entry.getValue())
+                  .build());
```

Comment:
nit I think addRequests would take a builder so you don't have to build here

Reply:
Done